src/sif/motorcyclecost.cc

```diff
@@ -487,15 +490,42 @@ Cost MotorcycleCost::TransitionCost(const baldr::DirectedEdge* edge,
         turn_cost += 0.5f;
     }
 
+    float seconds = turn_cost;
```

Comment:
moved the logic from autocost.cc to finish the logic for motor cycle, motor scooter, and truck costing.  Ref https://github.com/valhalla/valhalla/pull/2891  I also added tests as they were missing.